Readme.md

```diff
@@ -0,0 +1,81 @@
+> ### ⚠️ This is a proof of concept
+>
+> As this is a proof of concept,  it won't be supported by the k6 team.
+> It may also break in the future as xk6 evolves. USE AT YOUR OWN RISK!
+> Any issues with the tool should be raised [here](https://github.com/k6io/xk6-docker/issues).
+
+</br>
+</br>
+
+<div align="center">
+
+# xk6-docker
+A k6 extension for using of Docker CLI in testing. Built for [k6](https://github.com/loadimpact/k6) using [xk6](https://github.com/k6io/xk6).
+
+</div>
+
+## Build
+
+To build a `k6` binary with this extension, first ensure you have the prerequisites:
+
+- [Go toolchain](https://go101.org/article/go-toolchain.html)
+- Git
+
+Then:
+
+1. Download `xk6`:
+  ```bash
+  $ go get -u github.com/k6io/xk6
+  ```
+
+2. Build the binary:
+  ```bash
+  $ xk6 build --with github.com/k6io/xk6-docker
+  ```
+
+## Example
+
+```javascript
+import containers from 'k6/x/docker/containers';
+
+export default function () {
+  containers.listContainers().forEach((container) => {
+    console.log(container.id.substring(0, 10)));
+  })
+}
+```
+
+Result output:
+
+```bash
```

Comment:
plain might be preferable here given the weird syntax highlighting of the output?

Reply:
✅